core/src/main/scala/chisel3/package.scala

```diff
@@ -207,9 +207,6 @@ package object chisel3 {
     a.allElements
   }
   def getModulePorts(m: Module): Seq[Port] = m.getPorts
-  // Invalidate API - a DontCare element for explicit assignment to outputs,
-  //  indicating the signal is intentionally not driven.
-  val DontCare = chisel3.internal.InternalDontCare
```

Comment:
Something about this being a val caused type inference issues with the versions of things like [`WireDefault.apply` ](https://www.chisel-lang.org/api/3.4.1/chisel3/WireDefault$.html#apply[T%3C:chisel3.Data](t:T,init:chisel3.DontCare.type)(implicitsourceInfo:chisel3.internal.sourceinfo.SourceInfo,implicitcompileOptions:chisel3.CompileOptions):T)that accept `DontCare.type`. I don't know exactly what the change is, but the solution, to just get rid of the aliasing is cleaner anyway. It also solves https://github.com/chipsalliance/chisel3/issues